extension/scripts/global/functions/filters.css

```diff
@@ -92,7 +92,7 @@
 
 /* New Filters CSS Model */
 .tt-filter {
-	font-family: Arial;
+	font-family: Arial !important;

```

Comment:
If you need `!important` on this selector, you are probably doing something wrong.

Reply:
I added this as the Newspaper page has some other Serif font.

Reply:
Yeah, but we don't have the filter class there and I also like our filter being in the newspaper style.

Reply:
Did not know that. Will remove it.